Core/Util/DisArm64.cpp

```diff
@@ -508,6 +507,7 @@ static void DataProcessingRegister(uint32_t w, uint64_t addr, Instruction *instr
 			if (opn < 4 && Ra == 31) {
 				snprintf(instr->text, sizeof(instr->text), "%cmull x%d, w%d, w%d", sign, Rd, Rn, Rm);
 			} else {
+				const char* opnames[8] = { 0, 0, "maddl", "msubl", "smulh", 0, 0, 0 };
```

Comment:
while at it, can make this `static const char * const`

Reply:
check this fix in here: https://github.com/hrydgard/ppsspp/pull/18581/commits/2a31f8c6c05f8c9f1cb090d74f1cb170b829985e